taker-frontend/src/App.tsx

```diff
@@ -54,6 +55,15 @@ async function postCfdOrderRequest(payload: CfdOrderRequestPayload) {
     }
 }
 
+export async function postWithdraw(payload: WithdrawRequest) {
+    let res = await fetch(`/api/withdraw`, { method: "POST", body: JSON.stringify(payload) });
+    if (!res.status.toString().startsWith("2")) {
+        console.log(`Error${JSON.stringify(res)}`);
+        throw new Error("failed to withdraw: " + res.status + ", " + res.statusText);
```

Comment:
Your error is being swallowed because you are completely ignoring the body here. Check how `createErrorToast` is used in other places. I believe it has capabilities to parse an RFC7807 response which should give you the error.

Reply:
I use it the same as in other places (throw an error here and the caller catches and uses `createErrorToas`)
 but maybe that has never worked?

Reply:
🤦‍♂️
... `import createErrorToast from "../../../maker-frontend/src/components/ErrorToast";`

Reply:
@bonomat there's a copy in the `taker-frontend/src/components`! (I copied it there)

It works, as the other toasts for failed actions are displayed correctly.

Reply:
I know, I've imported the wrong one which did not work 😅